tools.rb

```diff
@@ -8,6 +8,8 @@ class MainCommand < Clamp::Command
 
   subcommand "cherry-picks", "Calculate needed cherry picks for a given release configuration", ToolBelt::Command::CherryPickCommand
   subcommand "changelog", "Generate changelog for a given release", ToolBelt::Command::ChangelogCommand
+  subcommand "branch", "Create and push the branch or tag for each repository as specified in the configuration", ToolBelt::Command::BranchCommand
+  subcommand "branch-docs", "Generate docs for a given release", ToolBelt::Command::BranchDocsCommand
```

Comment:
So one issue is this is very katello specific the way the code is written and this tool is used by at least Foreman and Katello for releases.

Reply:
Yeah, I realized this during development. I'm not really sure of a good solution there other than checking the project field or having maybe a project specific namespace (i.e. `Katello::BranchDocsCommand`).

Reply:
For now I [added a check on project](https://github.com/ehelms/tool_belt/pull/57/files#diff-ff264a62dc178b509ae2f522f3b5c129R27).